### PR TITLE
Feat: added cart management page

### PR DESCRIPTION
**Pull request by aryan-13**
Added the following:

- horizontal cards
- price details
- icon button
- responsiveness

Preview: https://deploy-preview-4--lunarstore.netlify.app/cart.html

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+                                <button class="btn-icon-secondary">
+                                    <i class="fa fa-minus"></i>
+                                </button>
+                                <span id="quantity">4</span>
+
+                                <button class="btn-icon-secondary">
+                                    <i class="fa fa-plus"></i>
+                                </button>
+                            </div>
+                            <a href="" class="btn btn-sm btn-primary">Move to Wishlist </a>
```
Comment (Hamdrive) [inline cart.html:98]: Add a link to href in anchor tag

**Review by Hamdrive — COMMENTED** [review]
Code looks good, would suggest to format code better to improve readability

File: cart.html (hunk 1 above)
Comment (aryan-13) [reply cart.html:98]: I haven't yet added the wishlist page

**Review by anshikaa12 — APPROVED** [review]
code looks good to merge!

**Pull request merged**